core/app/assets/js/ui/atoms/select.js

```diff
@@ -19,11 +19,11 @@ export default function Select( props ) {
 }
 Select.propTypes = {
 	className: PropTypes.string,
-	value: PropTypes.string,
 	onChange: PropTypes.func,
 	options: PropTypes.array,
 	elRef: PropTypes.object,
 	multiple: PropTypes.bool,
+	value: PropTypes.oneOfType( [ PropTypes.array, PropTypes.string ] ),
```

Comment:
`<option key={ option.value } value={ option.value }>`
How can it be an array?

Reply:
here is the documentation I used to for line 26
https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

```
  optionalUnion: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
    PropTypes.instanceOf(Message)
  ]),
```